osgeo_importer_prj/settings.py

```diff
@@ -80,5 +80,6 @@
 
 OSGEO_DATASTORE = 'datastore'
 OSGEO_IMPORTER_GEONODE_ENABLED = True
+OSGEO_IMPORTER_VALID_EXTENSIONS = ['shp','shx','prj','dbf','kml','geojson','json','tif','tiff','gpkg','csv','zip','xml','sld']
```

Comment:
I will offer a tidy for this: I'd typically want a space after commas and since there's so many items we might as well have the opening/closing brackets on their own lines